src/card.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p,
+h2,
+span {
+  margin: 0;
+}
+
```

Comment:
checklist p11 - create separate file for each BEM block (maybe you should separate settings reset and card block into two different files)

Reply:
done